src/sections/ui/markdown/MarkdownComponent.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import ReactMarkdown from 'react-markdown'
+
+interface Props {
+  markdown: string
+}
+
+const MarkdownComponent: React.FC<Props> = ({ markdown }) => {
+  const containerRef = React.useRef<HTMLDivElement>(null)
+
+  React.useEffect(() => {
+    if (containerRef.current) {
+      ReactDOM.render(<ReactMarkdown>{markdown}</ReactMarkdown>, containerRef.current)
+    }
+
+    return () => {
+      if (containerRef.current) {
+        ReactDOM.unmountComponentAtNode(containerRef.current)
+      }
+    }
+  }, [markdown])
+
+  return <div ref={containerRef} />
+}
+
+export default MarkdownComponent
```

Comment:
To keep the consistency across the application we should upgrade this syntax to react 18. Some functions are deprecated. Try with:

```suggestion
import ReactMarkdown from 'react-markdown'
import { useRef } from 'react'

interface Props {
  markdown: string
}

export function MarkdownComponent({ markdown }: Props) {
  const containerRef = useRef<HTMLDivElement>(null)

  return (
    <div ref={containerRef}>
      <ReactMarkdown>{markdown}</ReactMarkdown>
    </div>
  )
}
```